### PR TITLE
Improve content metadata field docs

### LINKED ISSUE #2347: Improve metadata field documentation
Hello,

I've been experimenting with Pelican for some weeks now. I enjoy it much more than Hugo for several reasons:
* It supports reStructuredText by default.
* Multi-language articles is easy to use, both as a writer and reader.
* The documentation is much clearer and detailed.

Still, I think some things should be reworked in the current documentation. For exemple, on the “*Writing content article*”:
* It should be clearer about how the “File metadata” syntax to use, depending of the chosen markup style (Markdown, reStructuredText, AsciiDoc)
* A table should list all the available metadata: their name, function, use and if they are mandatory or not.

Other ideas may be posted below this post.

Of course, I'm ready to help with a pull request. Just let me know where is the complete metadata list and I'll submit something to you.

Best regards.

### PR DESCRIPTION
**Pull request by justinmayer**
Fixes #2347

### CODE REVIEW

File: docs/content.rst [hunk 1]
```diff
@@ -71,22 +71,31 @@ Metadata syntax for Markdown posts should follow this pattern::
...
+``:author:``      ``Author:``      Content author, when there is only one
+``:authors:``     ``Authors:``     Content authors, when there are multiple
+``:summary:``     ``Summary:``     Brief description of content for index pages
+``:lang:``        ``Lang:``        Content language ID (``en``, ``fr``, etc.)
+``:translation:`` ``Translation:`` Is article is a translation of another
+                                   (``true`` or ``false``)
+``:status:``      ``Status:``      Content status:
+                                   ``draft``, ``hidden``, or ``published``
+``:template:``    ``Template:``    Name of template to use to generate content
+``:save_us:``     ``Save_us:``     Save content to this relative file path
```
Comment (oulenz) [inline docs/content.rst]: should be `save_as`

File: docs/content.rst [hunk 2]
```diff
@@ -71,22 +71,31 @@ Metadata syntax for Markdown posts should follow this pattern::
...
+``:tags:``        ``Tags:``        Content tags, separated by commas
+``:keywords:``    ``Keywords:``    Content keywords, separated by commas
+``:category:``    ``Category:``    Content category (one only — not multiple)
+``:slug:``        ``Slug:``        Identifier used in URLs and translations
+``:author:``      ``Author:``      Content author, when there is only one
+``:authors:``     ``Authors:``     Content authors, when there are multiple
+``:summary:``     ``Summary:``     Brief description of content for index pages
+``:lang:``        ``Lang:``        Content language ID (``en``, ``fr``, etc.)
+``:translation:`` ``Translation:`` Is article is a translation of another
+                                   (``true`` or ``false``)
```
Comment (oulenz) [inline docs/content.rst]: `Is article is` -> `Is content`

File: docs/content.rst [hunk 3]
```diff
@@ -71,22 +71,31 @@ Metadata syntax for Markdown posts should follow this pattern::
     This is the content of my super blog post.
 
 You can also have your own metadata keys (so long as they don't conflict with
-reserved metadata keywords) for use in your python templates. The following is
+reserved metadata keywords) for use in your templates. Following is
```
Comment (oulenz) [inline docs/content.rst]: I think `Following is` -> `This is` would parse easier

File: docs/content.rst (hunk 1 above)
Comment (justinmayer) [reply docs/content.rst]: Fixed in https://github.com/getpelican/pelican/pull/2620/commits/f0617a53dc253432077fe3acbcd8d8bd87f636ab

File: docs/content.rst (hunk 2 above)
Comment (justinmayer) [reply docs/content.rst]: Fixed in https://github.com/getpelican/pelican/pull/2620/commits/f0617a53dc253432077fe3acbcd8d8bd87f636ab

File: docs/content.rst (hunk 3 above)
Comment (justinmayer) [reply docs/content.rst]: Didn't quite agree that was clearer, so I found another phrasing that might be easier to parse in https://github.com/getpelican/pelican/pull/2620/commits/f0617a53dc253432077fe3acbcd8d8bd87f636ab.

File: docs/content.rst [hunk 4]
```diff
@@ -71,22 +71,31 @@ Metadata syntax for Markdown posts should follow this pattern::
...
-* `Summary`
-* `Template`
-* `Save_as`
-* `Url`
+================= ================ ============================================
+reStructuredText      Markdown                     Description
+================= ================ ============================================
+``:title:``       ``Title:``       Title of the article or page
+``:date:``        ``Date:``        Publication date (``YYYY-MM-DD HH:SS``)
+``:modified:``    ``Modified:``    Modification date (``YYYY-MM-DD HH:SS``)
```
[the remark this answers is not in the document]
Comment (justinmayer) [reply docs/content.rst]: Understood, but there isn't a lot of room in that table to explain that, and the contents of the docs in that link are nearly inscrutable to a normal human being. I prefixed the example syntax with `e.g.,` to at least hint that there are other formats available. If someone wants to make a follow-up PR to expand upon this and elaborate on the available date/time formats provided by `dateutil`, we'll get it merged. I can see how that would be helpful.

File: docs/content.rst [hunk 5]
```diff
@@ -71,22 +71,31 @@ Metadata syntax for Markdown posts should follow this pattern::
...
+``:tags:``        ``Tags:``        Content tags, separated by commas
+``:keywords:``    ``Keywords:``    Content keywords, separated by commas
```
[the remark this answers is not in the document]
Comment (justinmayer) [reply docs/content.rst]: Quite right. I added a note explaining it applies to HTML source content only.

File: docs/content.rst [hunk 6]
```diff
@@ -71,22 +71,31 @@ Metadata syntax for Markdown posts should follow this pattern::
...
+``:slug:``        ``Slug:``        Identifier used in URLs and translations
+``:author:``      ``Author:``      Content author, when there is only one
+``:authors:``     ``Authors:``     Content authors, when there are multiple
+``:summary:``     ``Summary:``     Brief description of content for index pages
+``:lang:``        ``Lang:``        Content language ID (``en``, ``fr``, etc.)
+``:translation:`` ``Translation:`` Is article is a translation of another
+                                   (``true`` or ``false``)
+``:status:``      ``Status:``      Content status:
+                                   ``draft``, ``hidden``, or ``published``
+``:template:``    ``Template:``    Name of template to use to generate content
```
[the remark this answers is not in the document]
Comment (justinmayer) [reply docs/content.rst]: Fixed in https://github.com/getpelican/pelican/pull/2620/commits/f0617a53dc253432077fe3acbcd8d8bd87f636ab

**Comment by justinmayer** [comment]
Many thanks to @oulenz and @avaris for the feedback on this PR. To the best of my knowledge, I addressed all the comments. Let me know if there's anything further I should change before merging.

**Pull request merged**